memcached-image/memcached-image.xsl

```diff
@@ -0,0 +1,117 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<xsl:stylesheet version="1.0" xmlns:xsl="http://www.w3.org/1999/XSL/Transform">
+<xsl:output method="xml" encoding="UTF-8" indent="yes"/>
+<xsl:template match="/">
+<xsl:comment> OBS-ExclusiveArch: aarch64 x86_64 </xsl:comment>
+<xsl:comment>
+  This file is generated,
+  don't manually modify this file,
+  please check README
+</xsl:comment>
+
+<image schemaversion="6.9" name="memcached-image" xmlns:suse_label_helper="com.suse.label_helper">
+
+  <description type="system">
+    <author>Thomas Bechtold</author>
+    <contact>tbechtold@suse.com</contact>
+    <specification>memcached container</specification>
+  </description>
+
+  <preferences>
+    <type image="docker">
+      <xsl:attribute name="derived_from">
+        <xsl:value-of select="param/image" />
+      </xsl:attribute>
+      <containerconfig
+          tag="latest"
```

Comment:
`latest` might make sense here, because this is tumbleweed, but for other cases I think we should avoid using latest because it creates a moving target. This also seems to be the recommendation [here](https://confluence.suse.com/display/maintenance/Container+Maintenance+Internals).

Reply:
agreed.